organizer-closing.slide

```diff
@@ -96,6 +96,8 @@ Information for each local Go communities.
 
 - Reach out on the mailing-list: https://groups.google.com/forum/#!forum/gobridge
 
+- Participate in the forum: https://forum.golangbridge.org/
```

Comment:
Pls change to:

"Participate in the Go Forum:"

:+1:  to merge after this change. Thanks!